addons/crm_claim/models/res_partner.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    claim_count = fields.Integer(compute='_compute_claim_count')
+
+    def _compute_claim_count(self):
+        partner_data = self.env['crm.claim'].read_group([('partner_id', 'in', self.ids)], [], groupby="partner_id")
+        result = dict((data['partner_id'][0], data['partner_id_count']) for data in partner_data)
+        for partner in self:
+            partner.claim_count = result.get(partner.id, 0)
```

Comment:
@rap-odoo  I have Question,
Lets guess partner id is 543302 and assign to partner.claim_count
so as per your code,
```
partner's claim_count is 543302
```
and if not then 0(zero)
really is it your meaning ?????????